ulog/README.md

```diff
@@ -22,13 +22,12 @@ import "go.uber.org/fx/ulog"
 func main() {
   // Configure logger with configuration preferred by your service
   logConfig := ulog.Configuration{}
-  logBuilder := ulog.Builder().WithConfiguration(&logConfig)
 
-  // Build ulog.Log from logBuilder
-  log := lobBuilder.Build()
+  // Build logger from logConfig object
+  logger, err := svc.logConfig.Build(zap.Hooks(ulog.Metrics(svc.metrics)))
```

Comment:
also `logConfig` isn't going to be exported, how are people going to access this? or maybe i just am mis-understanding what `svc`